app/src/main/java/com/capstone/weatherapp/WeatherRepository.kt

```diff
@@ -26,19 +26,24 @@ class WeatherRepository(private val db: WeatherDatabase) {
 
     suspend fun refreshCityList() {
         withContext(Dispatchers.IO) {
-            val cityList = WeatherApiClient.retrofitService.getCities(CITIES, APIKEY, UNITS)
-            db.cityDao().insert(cityList.convertToCityCacheList())
+            try {
+                val cityList = WeatherApiClient.retrofitService.getCities(CITIES, APIKEY, UNITS)
+                db.cityDao().insert(cityList.convertToCityCacheList())
+            } catch (e: Exception) {}
+            _cityListData.postValue(getNetworkCityList())
```

Comment:
we don't want this inside the catch block because if the try block code is sucessful, this gets skipped over. let's move this back to where it was so that it gets called both if the try block code succeeds or fails